doc/man7/verification.pod

```diff
@@ -0,0 +1,339 @@
+=pod
+
+=head1 NAME
+
+verification - X.509 certificate verification
+
+=head1 SYNOPSIS
+
+=for openssl generic
+
+ #include <openssl/x509_vfy.h>
```

Comment:
Why this include when the whole manual page talks about command line options and not flags from the header?

Reply:
This is because the page is also referred to by `doc/man3/X509_verify_cert.pod` and the options apply also at the lib level, though this could be more explicit.
I remember that about half a year ago the pretty redundant doc of the lib-level verification options were removed in favor of the CLI-oriented description that is being moved here.

Reply:
Meanwhile I've replaced that part.